File: Modul3/SOAL5/PRAK305-2410817120013-OktaviaLailaQodari.py
totaldetik = int(input())
jam = totaldetik // 3600
sisadetik = totaldetik % 3600
menit = sisadetik // 60
detik = sisadetik % 60

if jam >= 24:
 hari = jam // 24
 jam = jam % 24
 print(f"{hari} hari {jam:02d}:{menit:02d}:{detik:02d}")
else:
 print(f"{jam:02d}:{menit:02d}:{detik:02d}")

File: Modul3/SOAL5/PRAK305-2410817120013-OktaviaLailaQodari.c
#include <stdio.h>

int main () {
 int totaldetik, jam, menit, detik;
 scanf("%d", &totaldetik);

 jam = totaldetik / 3600;
 totaldetik %= 3600;
 menit = totaldetik / 60;
 detik = totaldetik % 60;

 if (jam >= 24) {
    int hari = jam / 24;
    jam %= 24;
    printf("%d hari:%02d:%02d:%02d\n", hari, jam, menit, detik);
 }
 else {
    printf("%02d:%02d:%02d", jam, menit, detik);
 }
return 0;
}